docs/source/models/other_models/pregnancy/gbd_2021_mncnh/index.rst

```diff
@@ -115,76 +114,90 @@ GBD has estimated the prevalence of pregnancy (as an intermediate variable for t
 
 Where :math:`ASFR` is the age-specific fertility rate, :math:`SBR` is the stillbirth to live birth ratio, and :math:`46/52` is the proportion of the year spent pregnant (40 weeks) and postpartum (6 weeks).
 
-.. todo::
-
-   Determine the threshold of gestational age at which a loss of pregnancy is classified as a stillbirth rather than miscarriage for the GBD covariate. Standard thresholds are 20 or 24 weeks.
-
 Vivarium Modeling Strategy
 ----------------------------
 
-We will model pregnancy as a characteristic of women of reproductive age in our simulations. We will inform the *incidence* of pregnancy using the age-specific fertility and stillbirth to live birth ratio covariates from GBD. We will inform the *duration* of pregnancy using the GBD 2021 exposure distribution of gestational age.
-
-.. image:: diagram.svg
+In this closed cohort, all simulants start the simulation at the beginning of a pregnancy.
```

Comment:
I significantly revised this page to more cleanly separate it from the pregnancy demography page. Basically, we are assuming here that all simulants are pregnant and any reference to non-pregnant simulants is confusing.